Corona-Warn-App/src/main/java/de/rki/coronawarnapp/transaction/RiskLevelTransaction.kt

```diff
@@ -135,6 +135,12 @@ object RiskLevelTransaction : Transaction() {
     // Injecting here will break Test
     private val riskLevelCalculation: RiskLevelCalculation = DefaultRiskLevelCalculation()
 
+    /**
+     * The maximal runtime of the Risk Level transaction
+     * In milliseconds
+     */
+    private const val RISK_LEVEL_TRANSACTION_TIMEOUT = 420000L
```

Comment:
420s,7min, doesn't the Ticket say 8min,480s?

If you change it, make it `480 * 1000L` a bit easier to read :)

Reply:
😄  Done.